packages/core/src/index.tsx

```diff
@@ -9,6 +17,14 @@ if (
 ) {
     registerServiceWorker();
 }
+const has_endpoint_url = checkAndSetEndpointFromUrl();
 
-// eslint-disable-next-line
-import App from 'App/app.jsx';
+// if has endpoint url, APP will be redirected
+if (!has_endpoint_url) {
```

Comment:
please check that is it possible to move this if condition inside app.tsx component
I mean inside AppWithoutTranslation component
and in case of any error we can throw it in the app.

Reply:
Hello @ako,

It could be possible, but it would mean that we move the render logic with it, which we don't want to do because:

i.) we would be back to the point where we need to import the file (using import './App/app.jsx') instead of the component App, which we don't want
ii.) We would violate the single responsibility principle which is bad practise.

But again, it is possible.